lib/midtrans_api/api/transaction/charge_echannel.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module MidtransApi
+  module Api
+    module Transaction
+      class ChargeEchannel < MidtransApi::Api::Base
+        PATH          = 'charge'
+        PAYMENT_TYPE  = 'echannel'
```

Comment:
is echannel type for mandiri ki? Please add comment for it if echannel for mandiri

Reply:
I already update the details at README.md file ya cent, please help to check